### ISSUE TITLE
Testing parameters for Semantic KITTI

### ISSUE THREAD

**Issue by YaraAlnaggar** [HuguesTHOMAS/KPConv-PyTorch#28]
Hello, 
When testing your network on Semantic KITTI and obtaining the the current score on the leaderboard, were the parameters values different from the current values in train and test files? like `val_radius` in train_SemanticKitti.py and other convolution parameters. I was trying to replicate your results and wanted to make sure that I use the same parameters that you have used. 
Thanks

**Comment by HuguesTHOMAS** [comment]
@YaraAlnaggar,

My results on SemanticKitti are quite old and were computed with the original tensorflow implementation. At the time this was just a test for me and I did not saved the parameters etc. 

From there, I did not really had time to do it again so the current parameters may not be the one that used before. From what I remember, I did not have the possibility to have a different radius for validation, so I used the same radius for training and test, and I used a rather small radius (a few meters, maybe 3 or 4) to help detect small classes like bikes/pedestrians.

I am sorry I can't help you more, but if you have time to experiment with different parameters, you might end up having even a better score than mine on the leaderboard.

Best,
@HuguesTHOMAS
